scripts/clang-format.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Install Intel FPGA SDK for OpenCL Pro Edition
+# https://fpgasoftware.intel.com/
+
+set -eu -o pipefail
+
+find . -regex '.*\.\(c\|cpp\|h\)' -exec clang-format -i "$@" {} +
```

Comment:
Could you use the initial `| sort | xargs` variant and call this script in the clang-format workflow, so it gets tested?

Without `--verbose` the effect is the same, but with the optional `--verbose` the list of files is sorted.

Reply:
I have changed it to the same command used in workflow, added it in workflow and added a test commit that test this script.